modules/setting/setting.go

```diff
@@ -601,6 +602,8 @@ please consider changing to GITEA_CUSTOM`)
 	ShowFooterVersion = Cfg.Section("other").Key("SHOW_FOOTER_VERSION").MustBool()
 	ShowFooterTemplateLoadTime = Cfg.Section("other").Key("SHOW_FOOTER_TEMPLATE_LOAD_TIME").MustBool()
 
+	UI.ShowUserEmailInExplore = Cfg.Section("ui").Key("SHOW_USER_EMAIL_IN_EXPLORE").MustBool()
```

Comment:
`MustBool` should take a boolean IIRC, and should default to `true` in this case to honor the ini-file :)